plugins/hydra_ray_launcher/hydra_plugins/hydra_ray_launcher/conf/__init__.py

```diff
@@ -188,9 +188,9 @@ class RayAWSLauncherConf:
     _target_: str = "hydra_plugins.hydra_ray_launcher.ray_aws_launcher.RayAWSLauncher"
 
     mandatory_install: PluginMandatoryInstallConf = PluginMandatoryInstallConf()
-    ray_init_cfg: Dict[str, Any] = field(default_factory=dict)
-    ray_remote_cfg: Dict[str, Any] = field(default_factory=dict)
-    ray_cluster_cfg: RayClusterConf = RayClusterConf()
+    ray_init: Dict[str, Any] = field(default_factory=lambda: {"address": None})
+    ray_remote: Dict[str, Any] = field(default_factory=dict)
+    ray_cluster: RayClusterConf = RayClusterConf()
```

Comment:
If we went with my proposal to have a ray node, this would be:
```python
@dataclass
class RayConfig:
    init: Dict[str, Any] = field(default_factory=lambda: {"address": None})
    ray_remote: Dict[str, Any] = field(default_factory=dict)
    cluster: RayClusterConf = RayClusterConf()

@dataclass
class RayLauncherConf:
  ray : RayConfig = RayConfig()
  ..

@dataclass
RayAWSLauncherConf
  ray : RayConfig = RayConfig()
  ..
```